src/components/alert/alertCenter.svelte

```diff
@@ -4,16 +4,25 @@
 
   type Action = {
     kind?: ButtonKind
-    text: string
     action: (alert: AlertInfo) => void
-  }
-
-  interface AlertOptions {
+  } & (
+    | {
```

Comment:
```suggestion
    {
```

surprised this syntax is valid :laughing:

Reply:
Yeah! I found it weird too, but that's what `prettier` did.

Reply:
For whatever reason, my local config for prettier is doing that. Not sure why it's not respecting the repo config. 🤔